istio/agent/vendor/golang.org/x/sys/unix/mkerrors.sh

```diff
@@ -471,7 +471,7 @@ echo '#include <signal.h>' | $CC -x c - -E -dM $ccflags |
 echo '// mkerrors.sh' "$@"
 echo '// Code generated by the command above; see README.md. DO NOT EDIT.'
```

Comment:
what's reason for change?

Reply:
during rename IDE updated it as well, will rollback.

Reply:
during rename IDE updated it as well, will rollback.